core/src/components/datetime/test/disabled/datetime.e2e.ts

```diff
@@ -0,0 +1,156 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not differ across
+ * modes/directions.
+ */
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
+  test.describe(title('datetime: disabled'), () => {
+    test('should not change value when date is clicked', async ({ page }) => {
```

Comment:
This test still passes even if I re-enable the button (and remove the pointer-events code). I recommend using the [toBeDisabled](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-disabled) assertion instead. With this approach we can verify that a button is disabled (and therefore cannot be activated) rather than testing the side effect of a button being disabled.